visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts

```diff
@@ -51,27 +51,13 @@ describe("nodeContextMenuController", () => {
 	}
 
 	function mockElement() {
-		element = jest.fn<Element>(() => {
-			return [
-				{
-					children: [
-						{
-							clientWidth: 50,
-							clientHeight: 100
-						}
-					]
-				}
-			]
-		})()
+		element = [{}] as any
+		element[0] = Object.assign(element[0], { children: [{ clientWidth: 50, clientHeight: 100 }] })
```

Comment:
```suggestion
		element = [{ children: [{ clientWidth: 50, clientHeight: 100 }] }] as unknown
```

Reply:
This is not working since element is missing some properties not defined on that object. You can resolve this by changing `as unknown` to `as any`

Reply:
Using `any` is not a good practice. I would in fact suggest to try to use the TypeScript strict mode at some point. But that's just me and you have to know how you want to have your code.